extension/app/ts/utils/typescript.ts

```diff
@@ -49,3 +49,7 @@ export function isObject(maybe: unknown): maybe is Object {
 export function assertIsObject(maybe: unknown): asserts maybe is Object {
 	if (!isObject(maybe)) throw new Error(`Expected object but got ${ typeof maybe }`)
 }
+
+export async function browserStorageLocalGet(keys: string | string[]) {
+	return (await browser.storage.local.get(keys)) as { [key: string]: unknown }
```

Comment:
Two minor style suggestions:
1. `Record<string, unknown>` instead of `{ [key: string]: unknown }`.  I believe they are synonymous, but I think `Record` looks a bit nicer and is a bit more clear.
2. You can get rid of the parenthesis by just changing the return type to `Promise<T>`.  I like this a bit better personally, but I can appreciate that it is longer and not everyone will agree.  I generally dislike a lot of parenthesis.
```suggestion
	return await browser.storage.local.get(keys) as Promise<Record<string, unknown>>
```

Reply:
It is worth noting that the values that can correctly be stored are limited to `boolean, string, number, [], and {}`.  Essentially, only stuff that can be JSON encoded which means no `bigint, Map, Set, etc.`.  I wonder if it is worth creating a type for this that is more specific, and possibly a similar helper function for `browserStorageLocalSet`, that is type constrained?
```ts
type JsonPrimitive = string | number | boolean
type JsonType = JsonPrimitive | Array<JsonPrimitive> | Record<string, JsonPrimitive>
```

Alternatively, we could perhaps override the types for `browser.storage.local.get` and `browser.storage.local.set` so we don't need the helper functions?  You'll have to lookup how to do type overrides (and if it is possible in this context), I always forget how to do it.

Reply:
I created issue about this. I would rather use helper functions than override types & function calls as it can make it hard to understand why it behaves differently on this repo than in some other